src/Midgard/CreatePHP/Mapper/ChainRdfMapper.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Midgard\CreatePHP\Mapper;
+
+
+use Midgard\CreatePHP\RdfChainableMapperInterface;
+use Midgard\CreatePHP\RdfMapperInterface;
+use Midgard\CreatePHP\Entity\PropertyInterface;
+use Midgard\CreatePHP\Entity\CollectionInterface;
+use Midgard\CreatePHP\Type\TypeInterface;
+use Midgard\CreatePHP\Entity\EntityInterface;
+
+use \RuntimeException;
+
+/**
+ * Looks at all registered mappers to find one that can handle objects.
+ *
+ * @package Midgard.CreatePHP
+ */
+class ChainRdfMapper implements RdfMapperInterface
+{
+    /**
+     * All the registered mappers
+     *
+     * @var RdfMapperInterface[]
+     */
+    private $mappers = array();
+
+    /**
+     * Mappers index by the spl_object_hash of objects created by CreatePHP
+     *
+     * @var RdfMapperInterface[]
+     */
+    private $createdObjects = array();
+
+    /**
+     * Register a mapper with a key. The key will be prefixed to all subjects.
+     *
+     * @param RdfChainableMapperInterface $mapper
+     * @param string $mapperKey
+     */
+    public function registerMapper(RdfChainableMapperInterface $mapper, $mapperKey)
+    {
+        $this->mappers[$mapperKey] = $mapper;
+    }
+
+    /**
+     * Get the mapper than can handle object.
+     *
+     * @param mixed $object
+     * @return RdfChainableMapperInterface
+     * @throws RuntimeException when no mapper can handle the object
+     */
+    protected function getMapperForObject($object)
+    {
+        foreach ($this->mappers as $mapper)
+        {
```

Comment:
if we follow symfony codestyle, this `{` would need to be at the end of the line above.